blockbase/inc/customizer/wp-customize-colors.php

```diff
@@ -121,21 +131,45 @@ function register_color_controls( $wp_customize, $palette ) {
 		}
 	}
 
+	function register_duotone_controls( $wp_customize ) {
+		$wp_customize->add_setting(
+			'duotone_control',
+			array(
+				'default'           => true,
+				'capability'        => 'edit_theme_options',
+				'sanitize_callback' => array( __CLASS__, 'sanitize_boolean' ),
+				'transport'         => 'postMessage', // We need this to stop the page refreshing.
+			)
+		);
+
+		$wp_customize->add_control(
+			'duotone_control',
+			array(
+				'type'    => 'checkbox',
+				'section' => $this->section_key,
+				'label'   => __( 'Apply these colors to the theme\'s images', 'blockbase' ),
```

Comment:
This description isn't quite accurate - it depends what blocks the duotone is already applied to. How about just "enable duotone"?